great_expectations/render/renderer/v3/suite_profile_notebook_renderer.py

```diff
@@ -198,7 +198,7 @@ def _add_user_configurable_profiler_cells(self) -> None:
     #     "random_seed": 43792,
     # },
     # numeric_columns_high_precision_rule={
-    #     "round_decimals": 4,   # parameters will be rounded to 4 decimal places for metrics that increase precision
+    #     "round_decimals": None,   # parameters will be rounded to the maximum number of decimal places for metrics that increase precision
```

Comment:
@NathanFarmer No, please let us leave it at 4 -- that was a good approach, meaningful for high precision.  Thank you.